analysis/active_analyses.R

```diff
@@ -609,19 +600,55 @@ for (c in cohorts) {
                          episode_event_threshold = episode_event_threshold,
                          covariate_threshold = covariate_threshold,
                          age_spline = TRUE,
-                         analysis = "sub_prior_parkinson_risk_false")
-  }
-    
+                         analysis = "sub_history_parkinson_risk_false")
   }
+}
 
 # Assign unique name -----------------------------------------------------------
 
 df$name <- paste0("cohort_",df$cohort, "-", 
                   df$analysis, "-", 
                   gsub("out_date_","",df$outcome))
 
-# Fix anxiety prior history variables ------------------------------------------
+# Remove covariate according to each outcome -----------------------------------
+print("Removing coviriates according to each outcome")
+
+  # Cognitive impairment symptoms
+
+  df$covariate_other <- ifelse(df$outcome=="out_date_cognitive_impairment_symptoms",gsub("cov_bin_history_cog_imp_sympt;", "", df$covariate_other),df$covariate_other)
+    
+  # Parkinson disease 
+
+  df$covariate_other <- ifelse(df$outcome=="out_date_parkinson",gsub("cov_bin_history_parkinson;", "", df$covariate_other),df$covariate_other)
+    
+  # Alzheimer 
 
+  df$covariate_other <- ifelse(df$outcome=="out_date_cognitive_impairment_symptoms",gsub("cov_bin_history_cog_imp_sympt;", "", df$covariate_other),df$covariate_other)
+  
+  # Vascular dementia 
+
+  df$covariate_other <- ifelse(df$outcome=="out_date_vascular_dementia",gsub("cov_bin_history_vascular_dementia;", "", df$covariate_other),df$covariate_other)
+
+  # Lewy body 
+
+  df$covariate_other <- ifelse(df$outcome=="out_date_lewy_body_dementia",gsub("cov_bin_history_lewy_body_dementia;", "", df$covariate_other),df$covariate_other)
```

Comment:
This is correctly implemented in terms of code but (using lewy body dementia as an example) we don't have a `cov_bin_history_lewy_body_dementia` in the analysis, just `any dementia`. I think it is best we discuss this at the meeting tomorrow.

Reply:
Updated.